mass-deployer/pkg/mass-deployer/types.go

```diff
@@ -0,0 +1,44 @@
+package deployer
+
+type Config struct {
+	NodeGroups []NodesGroup      `yaml:"node_groups"`
+	Vms        []Vms             `yaml:"vms"`
+	SSHKeys    map[string]string `yaml:"ssh_keys"`
```

Comment:
is that a map to be able to reference sshkeys by name? if not maybe it's better to convert to slice

Reply:
Yes, to reference sshkeys easily